app/controllers/standard_steps_controller.rb

```diff
@@ -3,7 +3,9 @@
 class StandardStepsController < StepsController
   def edit
     @step = step_class.new(
-      current_snap_application.attributes.slice(*step_attrs),
+      HashWithIndifferentAccess.
```

Comment:
What do you think about extracting a private method like

```ruby
def application_attributes
  HashWithIndifferentAccess.new(current_snap_application.attributes)
end
```

on which you could call `.slice(*step_attrs)`?

Not sure if that's better, just feels like this one controller action is doing a good amount of work.